mdata/aggmetric.go

```diff
@@ -631,3 +624,18 @@ func (a *AggMetric) gcAggregators(now, chunkMinTs, metricMinTs uint32) (uint32,
 	}
 	return points, stale
 }
+
+func (a *AggMetric) discardedMetricsInc(err error) {
+	var reason string
+	switch err {
+	case mdataerrors.ErrMetricTooOld:
+		reason = sampleOutOfOrder
+		metricsTooOld.Inc()
+	case mdataerrors.ErrMetricNewValueForTimestamp:
+		reason = newValueForTimestamp
+		discardedNewValueForTimestamp.Inc()
+	default:
+		reason = "unknown"
+	}
```

Comment:
from #1201 :

> Every point we receive should either be persisted or we should increment a counter to say we discarded 

thus in the unknown case, we should also increment a carbon counter.

Reply:
Shall we create a new catch-all sort of counter? `tank.discarded.unknown` perhaps?
(note that this case does not happen atm)

Reply:
Done